watertap/unit_models/MD/membrane_distillation_base.py

```diff
@@ -40,9 +43,24 @@
 )
 from watertap.core.util.initialization import interval_initializer
 
+
 __author__ = "Elmira Shamlou"
 
 
+class MDconfigurationType(Enum):
+    """
+    DCMD: Direct Contact Membrane Distillation
+    VMD: Vacuum Membrane Distillation
+    PGMD_CGMD: Permeate Gap or Conductive Gap Membrane distillation
```

Comment:
I feel like the user-facing options should be "PGMD" and "CGMD", even if they end up using the same code.

Reply:
That works as well. They must manually adjust the gap thermal conductivity to switch between the two

Reply:
Maybe this option can just be `GMD`, and in documentation, we describe as being applicable for PGMD or CGMD.

Reply:
@ElmiraShamlou - the one other change I think we should do is simplify the name of this config option to `GMD`. The other option I can think of is making two options: "PGMD" and "CGMD". 

However, I think it'd be easier and clearer to users if we just replaced all instances of `PGMD_CGMD` with `GMD`. Then, the documentation would simply describe GMD as representing permeate gap or conductive gap MD. 

The reason I want to do this now is because we will have less time/resources for making API adjustments or refactoring in general going forward, unless such work is deemed beneficial to meet the needs of practitioners and solve problems in industry.

Reply:
how about FGMD (Filled Gap Membrane Distillation)?

Reply:
@ElmiraShamlou Personally, I prefer GMD, which would still be described as "Permeate Gap or Conductive Gap Membrane distillation" in documentation everywhere. Later on, if we were to add air-gap, we'd add "AGMD" to distinguish. The only thing I have against "filled gap" MD is that it is not a commonly used term in the literature. Gap MD is generic enough that the user should be able to ascertain what he or she is modeling.

Reply:
@adam-a-a I've updated the name to GMD as requested, but I kept the description as PGMD and CGMD to avoid confusion with AGMD.